src/library/scala/util/Using.scala

```diff
@@ -148,10 +148,8 @@ object Using {
   def resource[R: Resource, A](resource: R)(body: R => A): A = {
     if (resource == null) throw new NullPointerException("null resource")
 
-    @inline def safeAddSuppressed(t: Throwable, suppressed: Throwable): Unit = {
-      // don't `addSuppressed` to something which is a `ControlThrowable`
-      if (!t.isInstanceOf[ControlThrowable]) t.addSuppressed(suppressed)
-    }
+    @inline def safeAddSuppressed(t: Throwable, suppressed: Throwable): Unit =
+      t.addSuppressed(suppressed)  // a no-op for ControlThrowable
```

Comment:
can actually just inline this below then

Reply:
Yes, a less-lazy person would do that. Actually just documenting the baby step, respecting the steps that came before.